### PR TITLE
Code Coverage Gem Added

### PR DESCRIPTION
**Pull request by sfawcett123**
Code coverage Gem added to allow for better reporting in Sonacloud

### CODE REVIEW

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1784-website.azurewebsites.net

File: Gemfile [hunk 1]
```diff
@@ -92,6 +92,7 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'rspec-rails', '~> 5.0'
   gem 'rspec-sonarqube-formatter'
+  gem 'simplecov'
```
Comment (leoapost) [inline Gemfile:95]: We can move this to the `:test` group and add `require: false`, so it won't be required everytimg bundler starts.

Comment (sfawcett123) [reply Gemfile]: Done - I havent a clue really, I just copied this from Get Into Teaching :)

**Review by leoapost — CHANGES_REQUESTED** [review]
Simplecov generates the coverage report in the `./coverage/` folder which will be picked up by git, but we don't need to commit it. 

This should do it
```
echo "coverage" >> .gitignore
```

**Comment by sfawcett123** [comment]
> Simplecov generates the coverage report in the `./coverage/` folder which will be picked up by git, but we don't need to commit it.
> 
> This should do it
> 
> ```
> echo "coverage" >> .gitignore
> ```

I hadn't thought about that because when I run it, it is on GitHub and the results are in a container, so are destroyed after the run, but I will add it because it maybe ran manually.

File: Gemfile (hunk 1 above)
Comment (leoapost) [reply Gemfile]: Leaving it there wouldn't cause any issues, but it's just a bit more efficient now :)

**Review by leoapost — APPROVED** [review]
Looks good

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1784-website.azurewebsites.net

**Pull request merged**